e2e/support/commands/overwrites/log.js

```diff
@@ -1,14 +1,40 @@
-Cypress.Commands.overwrite("log", (originalFn, message) => {
-  Cypress.log({
-    displayName: `--- ${window.logCalls}. ${message} ---`,
-    name: `--- ${window.logCalls}. ${message} ---`,
+Cypress.Commands.overwrite("log", (originalFn, text) => {
+  const logConfig = {
+    displayName: `=== ${window.logCalls}. ${text}`,
```

Comment:
`---`
![image](https://github.com/metabase/metabase/assets/31325167/f05b9e60-c8d8-442b-84ed-9951a20d4cd5)

`--->`
![image](https://github.com/metabase/metabase/assets/31325167/110393c1-7fdd-4fd7-aa34-3e2f8d657a1f)

`>>>`
![image](https://github.com/metabase/metabase/assets/31325167/e76ee78f-c98d-4b6d-8d76-501387e52e11)

or build your own and suggest it here :)

Reply:
I understand the prefix was there to make the logs stand out.
Colors do a much better job now, so I think we don't need the ascii art anymore :tada: 

```suggestion
    displayName: `${window.logCalls}. ${text}`,
```